book/src/lint_configuration.md

```diff
@@ -477,6 +477,11 @@ The maximum cognitive complexity a function can have
 
 ## `disallowed-macros`
 The list of disallowed macros, written as fully qualified paths.
+Fields:
+path (required): the fully qualified path to the macro that should be disallowed;
+reason (optional): explanation why this macro is disallowed;
+replacement (optional): suggested alternative macro;
+allow-invalid (optional, false by default): when set to true, it will ignore this entry if the path doesn't exist, instead of emitting an error.
```

Comment:
- We are in markdown, so we can use a proper list here by prefixing each element with "- ".
- The name of the field should be surrounded by backquotes (`path`).
- Nitpick: no need for a ';' or ' .' at the end if this is a proper list without full sentences
- Maybe `Fields:` should be `**Fields:**` for consistency with `**Default Value:**`